src/Attestation/AttestationCommand.php

```diff
@@ -8,9 +8,20 @@
 
 class AttestationCommand
 {
+    public function __construct()
+    {
+        $this->date = new \DateTime();
+        // Let’s say you did that 10 minutes ago, ok ?
```

Comment:
Pourquoi ?

Reply:
Hum, je ne suis pas sûr que la raison soit avouable ;)

Reply:
Ok pour ajouter la date dans l'objet, c'est tout à fait censé.
Not ok pour antidater... :wink: